src/publish/common/bundle.ts

```diff
@@ -24,8 +30,9 @@ export async function createBundle(
   for (const file of files.files) {
     const filePath = join(files.baseDir, file);
     if (Deno.statSync(filePath).isFile) {
-      const f = Deno.readTextFileSync(filePath);
-      manifestFiles[file] = { checksum: md5Hash(f) as string };
```

Comment:
I found that this was calculating the wrong checksum – it only works because Posit Connect doesn't validate it. `md5Hash` works with strings, but I couldn't find a way to get it to work properly using the contents of a file. Since Deno's crypto module supports MD5, there's probably no reason to keep `blueImpMd5` – but I'll leave that to a future PR.